securedrop/requirements/develop-requirements.txt

```diff
@@ -39,20 +35,17 @@ docker-pycreds==0.2.1     # via docker-py
 docopt==0.6.2             # via html-linter, template-remover
 docutils==0.14            # via botocore, sphinx
 dparse==0.4.1             # via safety
-enum34==1.1.6             # via astroid, cryptography, flake8
```

Comment:
so there are a bunch of packages being removed in this diff - if only the version of `requests` changed, this doesn't make sense to me